world-clock/src/main.ts

```diff
@@ -1,12 +1,6 @@
-import { ProjectorMixin } from '@dojo/framework/widget-core/mixins/Projector';
+import renderer from '@dojo/framework/widget-core/vdom';
+import { w } from '@dojo/framework/widget-core/d';
 import App from './widgets/App';
 
-// Create a projector to convert the virtual DOM produced by the application into the rendered page.
-// For more information on starting up a Dojo 2 application, take a look at
-// https://dojo.io/tutorials/002_creating_an_application/
-const Projector = ProjectorMixin(App);
-const projector = new Projector();
-
-// By default, append() will attach the rendered content to document.body.  To insert this application
-// into existing HTML content, pass the desired root node to append().
-projector.append();
+const r = renderer(() => w(App, {}));
+r.mount();
```

Comment:
Were you able to get this to build locally? There are a few errors with this app at the moment.

Reply:
Yeah I did try it and the build worked on travis, didn't it?

Reply:
@mwistrand I just tried again and all seems to be well.